CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Changes
 
+* [#113](https://github.com/clojure-emacs/orchard/issues/113) Add ability to skip functionality that works by altering the classpath
+  * You an opt in to this choice by setting `"-Dorchard.fetch-java-sources-via-dynapath=false"`.
```

Comment:
Btw, why don't we just shorted this to `orchard.use-dynapath`? After all the sources were its only use anyways.

Reply:
> Btw, why don't we just shorted this to orchard.use-dynapath? After all the sources were its only use anyways.
 
Good call! Done